### PR TITLE
TESTS: Clear derivatives dir before and after running tests

### PR DESCRIPTION
**Pull request by hoechenberger**

### CODE REVIEW

File: tests/run_tests.py [hunk 1]
```diff
@@ -98,16 +106,22 @@ def run_tests(test_suite):
...
+        clean_derivatives(os.environ['BIDS_ROOT'])
 
         # run the pipelines
         for pipeline in test_tuple[1::]:
             pipeline()
 
+        # Delete derivatives after testing is finished. Skip this step on
+        # circle, as we intend to retain the generated reports.
+        if os.environ['CI'] != 'true':
+            clean_derivatives(os.environ['BIDS_ROOT'])
```
Comment (agramfort) [inline tests/run_tests.py:121]: I think your changes to cricle.yml are enough

Comment (hoechenberger) [reply tests/run_tests.py:121]: I thought this here might be nice to have for running tests locally. Thoughts?

**Comment by agramfort** [comment]
for me I would add a "make clean" target in the makefile

>

**Comment by hoechenberger** [comment]
@agramfort Ok I've reverted the change to `run_tests.py`. Regarding the Makefile suggestion, I would look into this some other time.

**Pull request merged**